README.md

```diff
@@ -22,4 +58,15 @@ Setting the below parameter within the `frontends` object array will tell the pi
 
 The storage account name will remove the `-` and take the first `8` characters of the `name` property. For example if the `name` was set to `my-application-name` then the storage account would use `myapplic` and be called `hmctsmyapplicshutterprod`.
 
-You can then forward the website manually to the shutter by setting the `shutter: true` in the `azure-public-dns` repository.
\ No newline at end of file
+You can then forward the website manually to the shutter by setting the `shutter: true` in the `azure-public-dns` repository.
+
+### TLS
+By default it will not set any SSL so you will need to set the below to configure them.
+
+If you would like to use a certificate from the ACME Key Vaults managed by Platform Operations, then you can set add following properties:
+```terraform
+{
+  certificate_name = "{Certificate Name}"
+  ssl_mode         = "AzureKeyVault"
```

Comment:
I don't think this is correct, it will fetch from there without this, this would have the side effect of changing front door to use lets encrypt which we don't want by default
```suggestion
```

Reply:
Copy and paste issue, but yes corrected